src/mkdocs_include_markdown_plugin/process.py

```diff
@@ -112,6 +112,17 @@
     flags=re.VERBOSE | re.MULTILINE,
 )
 
+# Matched html image definition.
+# e.g. <img src="path/to/image.png" alt="alt-text">
+MARKDOWN_HTML_IMAGE_REGEX = re.compile(
+    r'''
+        <img
+        \s+
+        src="(\S+?)"    # src = $1
```

Comment:
This regex only matches `<img src="..."`, but fails with `<img alt="..." src="..."`. Can you take into account any number of attributes between `img` and `src`?

Reply:
Thank you for comment! Updated the script and test.